mautic/mautic
# The reply-to address of an email is the from address if the mailer is owner

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes [#11212](https://github.com/mautic/mautic/issues/11212) 



#### Description:


This PR fixes an issue with the reply-to email address when the option `mailer is owner` is set up.
#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a new contact or use an existing contact and assign an owner to the contact
3. Create a new email
4. Set up the option `mailer is owner` in the settings
5. Send the email to the contact
6. The owner's email address is used as the reply-to address. The default email address was before 




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11322"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
